python/mxnet/rnn/rnn_cell.py

```diff
@@ -788,6 +788,12 @@ def unpack_weights(self, args):
     def pack_weights(self, args):
         return _cells_pack_weights(self._cells, args)
 
+    def reset(self):
+        super(SequentialRNNCell, self).reset()
+        if hasattr(self, '_cells'):
```

Comment:
The constructor has `self._cells = []`. Shouldn't this condition always be true?